docsk8s/quick-start/ls-k8s-quick-start.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[ls-k8s-quick-start]]
+== Quick start
+
+WARNING: This documentation is still in development and may be changed or removed in a future release.
+
+This guide describes how to set up {ls} to monitor a Kubernetes environment. Kubernetes logs and metrics will be monitored by Filebeat and Metricbeat, respectively, processed through a Logstash pipeline, and then delivered into an Elasticsearch cluster in the Kubernetes environment.
```

Comment:
```suggestion
This guide describes how to set up {ls} to deliver Kubernetes log to {es}. Kubernetes logs will be monitored by Filebeat, processed through a Logstash pipeline, and then delivered into an Elasticsearch cluster in the Kubernetes environment.
```

- logstash does not monitor Kubernetes environment. It is more like a tunnel to pass the log from filebeat to elasticsearch
- metricbeat monitors logstash metrics only. It doesn't monitor Kubernetes in the current example.

Reply:
Ah, I see! :-)
I've change this intro text to:
> This guide describes how to set up Logstash to deliver Kubernetes logs to Elasticsearch. The logs will be monitored by Filebeat, processed through a Logstash pipeline, and then delivered into an Elasticsearch cluster in the Kubernetes environment.